front/app/containers/ProjectsShowPage/index.tsx

```diff
@@ -51,24 +52,15 @@ const Container = styled.main<{ background: string }>`
   flex: 1 0 auto;
   height: 100%;
   min-height: calc(
-    100vh - ${(props) => props.theme.menuHeight + props.theme.footerHeight}px
+    100vh - ${stylingConsts.menuHeight + stylingConsts.footerHeight}px
   );
   display: flex;
   flex-direction: column;
   align-items: center;
   background: ${(props) => props.background};
 
   ${media.tablet`
-    min-height: calc(100vh - ${({ theme: { mobileMenuHeight } }) =>
-      mobileMenuHeight}px - ${({ theme: { mobileTopBarHeight } }) =>
-    mobileTopBarHeight}px);
-  `}
-
-  ${media.phone`
-    min-height: calc(100vh - ${({ theme: { mobileMenuHeight } }) =>
-      mobileMenuHeight}px - ${({ theme: { mobileTopBarHeight } }) =>
-    mobileTopBarHeight}px);
-  `}
+    min-height: calc(100vh - ${stylingConsts.mobileMenuHeight}px - ${stylingConsts.mobileTopBarHeight}px);`}
```

Comment:
Is this fine, or do we prefer to use theme (in case of the unlikely event where we'd decouple stylingConsts from theme)?

Reply:
I think it's fine to use the styling constants in this case 👍 The code feels easier to read and understand with them.